icons/pug/pug-original.svg

```diff
@@ -0,0 +1 @@
5.708 41.7s2.697 7.4 8.691 10.5c0 0-9.39-1.1-8.691-10.5z" class="st3"/><path id="path22" stroke-width="1" d="M78.624 55.5s-5.894-6.2-13.787-6.4h.2c-7.992.2-13.786 6.4-13.786 6.4 6.893-4.8 12.787-4.7 13.786-4.7-.1 0 6.694-.1 13.587 4.7z" class="st3"/><path id="path23" stroke-width="1" d="M71.73 42.5s-2.996-4.2-6.992-4.3h.2c-2.998.1-6.894 4.3-6.894 4.3 3.397-3.3 6.893-3.2 6.893-3.2s3.297-.1 6.794 3.2z" class="st3"/><path id="path24" stroke-width="1" d="M37.164 73.2s-4.695 2.3-8.092.9h-.1c-2.997-1.7-4.496-6.8-4.496-6.8s2.998 9 12.688 5.9z" class="st3"/><path id="path25" stroke-width="1" d="M91.911 73.2s4.696 2.3 8.092.9c3.996-1.7 4.596-6.8 4.596-6.8s-2.997 9-12.688 5.9z" class="st3"/><path id="path26" stroke-width="1" d="M42.559 41.2c2.597-.5 6.893-.6 10.29.5 4.296 1.5.8 7 1.698 7.3.9.3 2.098-3.8 10.09-3.4 8.093.4 8.992 4 10.09 3.4 1.1-.6-1.098-10 10.99-7.8 0 0-12.688-3.4-12.088 5.8 0 0-7.293-5.6-17.483-.6.1 0 2.697-8.6-13.587-5.2z" class="st3"/><path id="path27" stroke-width="1" d="M86.816 41.2c.2 0 .3.1.4.1.1 0-.1-.1-.4-.1z" class="st3"/><path id="path28" stroke-width="1" d="M86.816 41.2Z" class="st3"/><path id="path29" stroke-width="1" d="M39.062 28.9s-10.79 13.6-12.388 18.8c-1.598 5.3-2.797 27-4.196 30.1l-4.995-21.4 9.191-22.3Z" class="st3"/><path id="path30" stroke-width="1" d="M89.813 28.9s10.79 13.6 12.388 18.8c1.599 5.3 2.798 27 4.196 30.1l4.995-21.4-9.19-22.3z" class="st3"/><path id="path31" stroke-width="1" d="M89.314 28.9s11.588 9.7 14.985 20.9c3.397 11.2 1.998 24.8 4.596 26.5 3.696 2.4 7.892-11.9 9.29-13.4 2.199-2.4 9.492-8.5 9.991-9.6.5-1.1-14.786-17.8-21.48-21.1-8.091-3.8-18.082-4.1-17.382-3.3Z" class="st7"/><path id="path32" stroke-width="1" d="M99.204 34.9s13.687 17.5 13.487 39.3l5.495-11.2c-.1 0-4.896-14.3-18.982-28.1z" class="st8"/><path id="path33" stroke-width="1" d="M39.062 28.9s-11.588 9.7-14.985 20.9C20.68 61 22.079 74.6 19.48 76.3c-3.696 2.4-7.892-11.9-9.29-13.4C7.991 60.5.7 54.4.2 53.3c-.5-1.1 14.786-17.8 21.48-21.1 8.091-3.8 18.082-4.1 17.382-3.3Z" class="st7"/><path id="path34" stroke-width="1" d="M29.172 34.9S15.485 52.4 15.685 74.2L10.29 63s4.796-14.3 18.882-28.1z" class="st8"/><path id="path35" stroke-width="1" d="M21.78 74.6s.998 5.4 2.597 7.1c1.598 1.7.5-1.3.5-1.3s-1.7-.9-1.4-7.8c.3-6.9-1.698 2-1.698 2z" class="st3"/><path id="path36" stroke-width="1" d="M106.997 74.6s-1 5.4-2.598 7.1c-1.598 1.7-.5-1.3-.5-1.3s1.699-.9 1.4-7.8c-.3-6.9 1.698 2 1.698 2z" class="st3"/><ellipse id="circle36" cx="54.447" cy="70.5" stroke-width="1" class="st8" rx=".799" ry=".8"/><ellipse id="circle37" cx="49.852" cy="75.3" stroke-width="1" class="st8" rx=".799" ry=".8"/><ellipse id="circle38" cx="48.353" cy="70.5" stroke-width="1" class="st8" rx=".799" ry=".8"/><ellipse id="circle39" cx="73.929" cy="70.5" stroke-width="1" class="st8" rx=".799" ry=".8"/><ellipse id="circle40" cx="78.524" cy="75.3" stroke-width="1" class="st8" rx=".799" ry=".8"/><ellipse id="circle41" cx="80.023" cy="70.5" stroke-width="1" class="st8" rx=".799" ry=".8"/></svg>
```

Comment:
the style tag should be replaced with the relevant attributes.
So this `<style id="style1" type="text/css">.st1{fill:#efcca3}.st3{fill:#ccac8d}.st4{fill:#fff}.st7{fill:#56332b}.st8{fill:#442823}.st9{fill:#7f4a41}.st10{fill:#331712}</style>`

Should be replaced with fill attributes on each path. That way you should also be able to remove the classes and id's, making the icon a bit smaller in size

Reply:
hopefully d0bd6e3 resolves!